rest-api/README.md

```diff
@@ -0,0 +1,329 @@
+
+## List Offerings
+
+### Description
+Used to fetch offerings from a PFI
+
+
+### Endpoint
+`GET /offerings`
+
+### Authentication
+No authentication required.
+
+### Authorization
+No authorization required. Offerings are publicly accessible
+
+### Query Params
+| Param              | Description                                                                                                                                                        |
+| ------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `baseCurrency`     | The primary currency listed in a currency pair, representing the unit being traded. **This is the currency that Alice will be _receiving_**                        |
+| `quoteCurrency`    | The secondary currency listed in a currency pair, indicating its value in relation to the base currency. **This is the currency that the PFI will be _receiving_** |
+| `payinMethodKind`  | The payin method Alice wishes to use to provide quote currency                                                                                                     |
+| `payoutMethodKind` | The payout method Alice wishes to use to receive base currency                                                                                                     |
+| `id`               | Query for a specific offering                                                                                                                                      |
+
+
+### Response
+| Status             | Body                  |
+| ------------------ | --------------------- |
+| `200: OK`          | `Offering[]`          |
+| `400: Bad Request` | `{ errors: Error[] }` |
+
+---
+
+# Threads
+A thread is a series of linked tbDEX messages between Alice and a PFI for a single exchange. A thread can be created by submitting an RFQ.
+
+## Submit RFQ
+
+### Endpoint
+`POST /threads/:thread_id/rfq`
+
+### Authentication
+Refer to [Signature Verification Section]() of the tbDEX spec  
+### Authorization
+No Authorization required to submit an RFQ
+
+### Request Body
+> [!IMPORTANT]
+> See RFQ structure [here](../README.md#rfq-request-for-quote)
+
+### Response
+| Status             | Body                  |
+| ------------------ | --------------------- |
+| `202: Accepted`    | N/A                   |
+| `400: Bad Request` | `{ errors: Error[] }` |
+
+### Errors
+| Status | Description            |
+| ------ | ---------------------- |
+| 400    | Validation error(s)    |
+| 400    | Failed Signature Check |
+| 404    | Thread not found       |
+| 409    | RFQ already exists     |
+| 409    | Thread already exists  |
+
+## Get Quote
+### Endpoint
+`GET /threads/:thread_id/?messageType=quote`
```

Comment:
this is also descoped, because `GET /threads/:thread_id` overall is descoped below, yea?